.gitignore

```diff
@@ -3,3 +3,6 @@
 .python-version
 dist
 fastdtw.egg-info
+build
+*.c
```

Comment:
It is customary, at least in mature projects, to track generated C code. This way, Cython is not a made a build-dependency, and the build is unaffected even when Cython, whose development is still very active and somewhat unstable, changes on some odd day.